README.md

```diff
@@ -24,15 +24,12 @@ Please consult the [LORIS Wiki Setup Guide](https://github.com/aces/Loris/wiki/S
 
 # Prerequisites for Installation
 
- * LINUX (supported on Ubuntu 14+ and [CentOS 7](https://github.com/aces/Loris/blob/master/README.CentOS6.md))
  * Apache **2.4** or higher
  * MySQL 5.7
  * PHP <b>7.2</b> or higher
  * Package manager (for LINUX distributions)
- * Composer : should be run with --no-dev option
 
 <b>Important:</b>
- * If you are upgrading your LORIS, you'll also want to upgrade to both PHP 7.2 and MySQL 5.7.
```

Comment:
This still seems worth having.

Reply:
You don't think it's redundant given that we have a bulleted list of prereqs above?

Reply:
True, that makes sense.

Reply:
But I don't think "Important" is a very good title anymore. "Note" might be more accurate. If you don't specify --no-dev, you just get more dependencies installed.